src/material/schematics/ng-update/migrations/legacy-components-v15/constants.ts

```diff
@@ -41,6 +41,19 @@ export const MDC_IMPORT_CHANGES = COMPONENTS.map(component => ({
   new: `@angular/material/${component}`,
 }));
 
+export const CUSTOM_TS_SYMBOL_RENAMINGS = [
+  {old: 'ProgressAnimationEnd', new: 'LegacyProgressAnimationEnd'},
```

Comment:
I wonder if we can create a test using the TS AST checks all of the legacy entrypoints and:
- makes sure all the exported symbols contain "legacy" in some form
- verifies that for each exported symbol there is an equivalent in the non-legacy entry point that matches `legacySymbol.replace(/legacy[-_]?/i, '')`
- ensures that this map has an entry for each legacy symbol

Not something that needs to happen for this PR, but I think a test like that (if feasible) would help make sure we really caught everything